x/upgrade/internal/keeper/keeper.go

```diff
@@ -59,7 +59,7 @@ func (k Keeper) ScheduleUpgrade(ctx sdk.Context, plan types.Plan) sdk.Error {
 	return nil
 }
 
-func (k Keeper) getDoneHeight(ctx sdk.Context, name string) int64 {
+func (k Keeper) GetDoneHeight(ctx sdk.Context, name string) int64 {
```

Comment:
Can we add a simple godoc for this, please?

Reply:
Done

Reply:
Updated the godoc